Android/GoogleAndroidVending/docs/Licensing.md

```diff
@@ -0,0 +1,209 @@
+# Application Licensing
+
+[![NuGet](https://img.shields.io/nuget/vpre/Xamarin.Google.Android.Vending.Licensing.svg)][3]
+[![NuGet](https://img.shields.io/nuget/dt/Xamarin.Google.Android.Vending.Licensing.svg)][3]
```

Comment:
[1] not [3]

\[!\[NuGet](https://img.shields.io/nuget/vpre/Xamarin.Google.Android.Vending.Licensing.svg)][1]
\[!\[NuGet](https://img.shields.io/nuget/dt/Xamarin.Google.Android.Vending.Licensing.svg)][1]